README.md

```diff
@@ -44,77 +45,55 @@ $ git clone https://github.com/nvim-treesitter/nvim-treesitter.git
 
 ## Adding parsers
 
-Treesitter is using a different _parser_ for every language. It can be quite a pain to install, but fortunately `nvim-treesitter`
+Treesitter uses a different _parser_ for every language. It can be quite a pain to install, but fortunately `nvim-treesitter`
 provides two command to tackle this issue:
-  - `TSInstall` to install one or more parser. You can use `TSInstall all` to download all parsers.
-  - `TSInstallInfo` to know which parser is installed.
 
-Let's say you need parsers for `lua`, `c`, and `python`, this is how you do with these commands:
+- `TSInstall {language}` to install one or more parsers.
+  `TSInstall <tab>` will give you a list of supported languages, or select `all` to install them all.
+- `TSInstallInfo` to know which parser is installed.
 
-```vim
-:TSInstall c
-Downloading...
-Compiling...
-Treesitter parser for c has been installed
+Let's say you need parsers for `lua`, this is how you do with these commands:
```

Comment:
This example was taking a lot of space before having the configuration, so I cut it a little.